src/engraving/dom/bsymbol.cpp

```diff
@@ -123,7 +123,11 @@ EngravingItem* BSymbol::drop(EditData& data)
     EngravingItem* el = data.dropElement;
     if (el->isSymbol() || el->isImage()) {
         el->setParent(this);
-        PointF p = data.pos - pagePos() - data.dragOffset;
+        PointF p = data.pos - abbox().topLeft() - data.dragOffset;
```

Comment:
pagePos() sometimes returns a position that does not visually line up with the top of the symbol